packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -49,19 +51,19 @@ const isMonth = (possiblyMonthValue: any, presentYear: number): boolean => {
 };
 
 const isArrayOfType = (
-  data: Datum[] | Datum[][] | TypedArray,
+  data: Datum[] | Datum[][] | TypedArray | undefined,
```

Comment:
>undefined [](http://example.com/codeflow?start=43&length=9)

why can data be undefined?

Reply:
lets address this comment in current PR. Rest can be taken as followup refactoring.

Reply:
I have updated this function to handle cases where data might be undefined. For example, on line 92, 
https://github.com/microsoft/fluentui/blob/74998be29d2d6afe1142a55364cfeb60ea1eafc7/packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts#L92
data.y could be undefined, so there's no need to add a separate check for that.

Ideally, the argument should have the type any (similar to Array.isArray) since it will always return false if the argument is not an array.

Reply:
but data cannot be undefined if it is valid? Lets discuss on this and close today.

Reply:
As discussed offline, renamed argument to avoid confusion.